dids/src/main/kotlin/web5/sdk/dids/methods/key/DidKey.kt

```diff
@@ -111,6 +112,15 @@ public class DidKey(uri: String, keyManager: KeyManager) : Did(uri, keyManager)
       return DidKey(did, keyManager)
     }
 
+    /**
+     * Instantiates a [DidKey] instance from a "did:key" DID URI, and validates that the associated key material exists
+     * in the provided [keyManager].
+     */
+    override fun load(did: String, keyManager: KeyManager): DidKey {
```

Comment:
nit: on ln 62 I think the first argument is called `uri`

Reply:
Fixed